syncthing-bep/src/main/kotlin/net/syncthing/java/bep/connectionactor/HelloMessageHandler.kt

```diff
@@ -24,6 +24,7 @@ import java.io.DataInputStream
 import java.io.DataOutputStream
 import java.io.IOException
 
+//TODO This Logger is named differently. Why? Make Consistent?
```

Comment:
This TODO I'd like to address in this commit. Comment describes my questions.

Reply:
I'm not sure why. yes, make it consistent

Reply:
Alright, I need to talk through this one with you.
This one is unique in that it's not contained in a `class { ...}` or `object { ... }` block.
`private val LOGGER = LogManager.getLogger(HelloMessageHandler::Class.java)` triggers an error where `HelloMessageHandler` is an unresolved reference. Not sure if I need to make this whole script a class, or how to ask the right question here.